### DISCUSSION TITLE
auto-expand the Filter Context section for CALCULATE/CALCULATETABLE() in Debugger Evaluation Context Window

### DISCUSSION THREAD

**Discussion by edhans** [TabularEditor/TabularEditor3#1040] [Ideas]
**Is your feature request related to a problem? Please describe.**
When CALCULATE is used, a new FILTER CONTEXT is created, and the Evaluation Context window shows this, but it is always collapsed. Here is is after I've opened it. This one didn't modify the filters, because I wasn't filtering the date column in the date table

<img width="946" alt="image" src="https://github.com/TabularEditor/TabularEditor3/assets/45298358/f7e40e9c-0967-4df4-9dc2-e112a5266076">

Here is the measure again with this variable filtering on the Date field, and now my filters on Month, Month Name, and Year are removed. But the reason they are removed are hidden because the Filter Context section is collapsed.
<img width="942" alt="image" src="https://github.com/TabularEditor/TabularEditor3/assets/45298358/7bc070c8-4b53-4f96-935f-d9b4fe1ed1b2">

ANd that is because the engine adds an implict ALL to the date table since I am filtering on the Date field.
<img width="373" alt="image" src="https://github.com/TabularEditor/TabularEditor3/assets/45298358/28cb845f-0f4f-45f7-8177-e3a1a035963b">

That is useful info. Why is it hidden/collapsed by default? It should be expanded. In fact, trying hard to envision a place where I would want anything in the Evaluation Context window collapsed by default.

**Comment by DBojsen** [comment]
Hi @edhans 

Makes a lot of sense - I have added a backlog item for this.

Best Regards

David - Tabular Editor 3 Support
